source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,23 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+You can watch for changes in MongoDB by using the `watch()
```

Comment:
Could you specify what targets you can watch() either in this sentence or elsewhere in the guide? I wasn't able to find this info in your description and had to reference the server manual:
https://docs.mongodb.com/manual/changeStreams/#watch-collection-database-deployment

Reply:
Changed to:

"You can watch for changes in MongoDB, such as changes to a collection, database, or deployment, by using the watch() method."